tsnet/tsnet.go

```diff
@@ -108,21 +112,28 @@ func (s *Server) start() error {
 		s.hostname = prog
 	}
 
-	s.dir = s.Dir
-	if s.dir == "" {
+	s.rootPath = s.Dir
+	if strings.HasPrefix(s.rootPath, "mem:") && !s.Ephemeral {
+		return fmt.Errorf("in-memory store is only supported for Ephemeral nodes")
+	}
+	switch {
+	case strings.ContainsRune(s.rootPath, ':'):
```

Comment:
Just Contains, not ContainsRune.

But this needs a comment and I didn't see an answer to my Windows concern.

Reply:
I was adding tests so missed this bit, removed this. This was the last place left where I introduced it, and now it's also removed.